xds/src/main/java/io/grpc/xds/internal/sds/FileBasedPluginCredential.java

```diff
@@ -77,6 +78,19 @@
     secretData = buildDataSourceFromConfigStruct(value.getStructValue());
   }
 
+  FileBasedPluginCredential(
```

Comment:
Just wondering - what is the need to keep both the v2 and v3 versions? When everything is moved to v3 are we going to remove the v2 version of the constructor?

Reply:
I just kept the existing code unchanged, otherwise the whole class will be almost empty after removing v2 code and with v3 code unimplemented. We don't need v2 version for this class after migration.

Reply:
ok